src/components/onboarding/OnboardingStepper.vue

```diff
@@ -0,0 +1,158 @@
+<script lang="ts">
+/**
+ * OnboardingStepper Component
+ *
+ * @description * Use this component to display an onboarding widget.
+ *
+ * @events
+ * - `update:modelValue`: Emitted as a part of v-model structure.
+ *
+ * @components
+ * - `FormInviteFriends`: Component to render form for inviting friends.
+ *
+ * @example
+ * <onboarding-stepper />
+ *
+ * @see [Figma Design](https://www.figma.com/file/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?type=design&node-id=4858%3A105047&mode=dev)
+ */
+
+// libraries
+import { QStepper } from 'quasar';
+import { defineComponent, ref } from 'vue';
+
+// components
+import FormInviteFriends from 'components/form/FormInviteFriends.vue';
+
+export default defineComponent({
+  name: 'OnboardingStepper',
+  components: {
+    FormInviteFriends,
+  },
+  setup() {
+    const step = ref<number>(1);
+    const stepper = ref<InstanceType<typeof QStepper> | null>(null);
+
+    return {
+      step,
+      stepper,
+    };
+  },
+});
+</script>
+
+<template>
+  <q-stepper
+    animated
+    keep-alive
+    v-model="step"
+    ref="stepper"
+    color="primary"
+    header-class="hidden"
+    :style="{ 'max-width': '800px' }"
```

Comment:
We don't need style binding `:style` here,  instead of use simple inline style `style=`, please.

Reply:
Updated.